rero_ils/modules/libraries/api.py

```diff
@@ -90,6 +90,29 @@ def get_organisation(self):
         from ..organisations.api import Organisation
         return Organisation.get_record_by_pid(self.organisation_pid)
 
+    def get_address(self, address_type):
```

Comment:
This function should be much easier if we can work directly with `address`, `shipping` and `billing` or this could be handled  directly by the `LibraryAddressType` class.
In one line this could be:
```python
return self.get('acquisition_settings', {}).get(f'{address_type}_informations', {}).get('address')
```

Reply:
The structure of addresses are not the same and fields where to found it is different between main address and acq addresses.
The main address could be found using `self.get('address')` ; acq addresses could be found using your way